src/parser/jahia_site.py

```diff
@@ -289,23 +289,23 @@ def parse_breadcrumb(self):
 
         for language, dom_path in self.export_files.items():
             dom = Utils.get_dom(dom_path)
+            self.breadcrumb_url[language] = []
+            self.breadcrumb_title[language] = []
 
             breadcrumb_links = dom.getElementsByTagName("breadCrumbLink")
-            nb_found = len(breadcrumb_links)
-            if nb_found != 1:
-                logging.warning("Found %s breadcrumb link(s) instead of 1", nb_found)
-                if nb_found == 0:
-                    continue
+            if len(breadcrumb_links) == 0:
+                continue
             breadcrumb_link = breadcrumb_links[0]
```

Comment:
Et je vois aussi ça maintenant (désolé, ça m'a échappé avant XD).
Elle ne sert plus à rien cette ligne non? vu que juste après tu démarre une boucle `for` qui écrase direct la variable `breadcrumb_link`

Reply:
Ah effectivement, my bad. Corrigé.